### ISSUE TITLE
Updating challenges with links to guide articles

### ISSUE THREAD

**Issue by nsuchy** [freeCodeCamp/freeCodeCamp#15918]
As the guides project has been released it's time to starrt updating the challenges to include links to the guide articles rather than forum posts where applicable. This issue should serve as the discussion place for these changes.
****
**Tagged Maintainers:** @QuincyLarson @Bouncey

**Comment by systimotic** [comment]
I'll link to my comment about this: https://github.com/freeCodeCamp/guides/issues/145#issuecomment-327846080
> I'm not sure if having the hint buttons link to the guides is a good idea. They require some work to get up to par with the forum. Example: http://guide.freecodecamp.org/certificates/dna-pairing/

Having the challenges link to the Guide would be great, but as you can see from the link, some things have to be fixed before we can do this.

**Comment by QuincyLarson** [comment]
@systimotic You're right - some of these look pretty noisy.

@systimotic @nsuchy Would either of you be interested in going through and helping clean these up first before we switch to them as the replacement for the forum articles?

**Comment by nsuchy** [comment]
@QuincyLarson Yes I'm interested. Is there a list of articles that need cleanup?

**Comment by QuincyLarson** [comment]
@nsuchy Awesome! This is a big undertaking. We need to double check every single article in https://guide.freecodecamp.org/certificates

You may be able to solve some of these with find-replace like logic, but you'll want to make sure all of them look good locally before opening the pull request.

This would be a massive help for us, and help us shift from the forum to the guide for hints, massively expanding the amount of attention the guide gets.

**Comment by nsuchy** [comment]
@QuincyLarson I'll need to rebase my guides repo and open a new branch. To confirm you would like me to fix the design, and any information that may be inaccurate or unclear?

**Comment by QuincyLarson** [comment]
@nsuchy Yes - you can branch off of the guide repo and work through the files in that folder, eliminating unnecessary elements at the tops and bottoms of the article caused by bad imports. When you (or someone else) has time later, you can worry about accuracy and clarity of the text as a separate PR.

**Comment by systimotic** [comment]
@QuincyLarson I just submitted two PR's that should help with this a little bit.

I'm a bit of a control freak and an extreme perfectionist, so working on the Guide is difficult for me, as I can't control the quality of the content in it. That's also why I haven't been doing a lot of QA of the content: there are too many things I would want to change. 
Instead, I'm trying to focus my time on the places where I can make the largest improvement.

From my point of view, there is still lots and lots of work to be done before switching to the Guide version is a good idea, but realistically, the only real issue remaining is https://github.com/freeCodeCamp/guides/issues/587. It contains links to some examples, all in the certificate section.

There are 261 guides in this one section, so reviewing all of them manually and fixing the code blocks would be a large task. I don't know how we could do this in an automated way. Unless somebody is up to go through all of these files manually, we'll either have to find a solution at the plugin level or disable syntax highlighting.

**Comment by QuincyLarson** [comment]
@systimotic My understanding is that @nsuchy is going through these manually. In this case, it's a one-time process, and I don't think it's worth trying to come up with an automated way to do this, as each of these articles will still need to be QA'd. There's no escaping the fact that a human will have to manually look at this.

@nsuchy how is this process coming along?

**Comment by QuincyLarson** [comment]
@nsuchy How is this process coming along? I noticed you've been actively QAing [PRs related to this](https://github.com/freeCodeCamp/guides/pull/8078), but I'm just curious how close this is to completion overall.

**Comment by nsuchy** [comment]
I've largely been working more on writing content, I've QAed a few PRs that have fixed a lot of issues. I'll open another branch and check on what still needs to be done and will update you soon. Thanks.

**Comment by QuincyLarson** [comment]
@nsuchy Great - thanks for the update. Yes - I will do what I can to help get your PRs merged in as fast as possible. Just @-mention me :)

**Comment by huyenltnguyen** [comment]
Hi @nsuchy, I just discovered this issue. It looks like a massive task for one person. How is your progress? Let me know if I can give you a hand.

**Comment by QuincyLarson** [comment]
I'm closing this issue as stale since it hasn't been active lately. If you think this is still relevant to the newly updated platform, please explain why, then reopen it.
